service/internal/crdcache/cache.go

```diff
@@ -0,0 +1,84 @@
+package crdcache
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/giantswarm/app/v5/pkg/crd"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	gocache "github.com/patrickmn/go-cache"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+)
+
+const (
+	expiration = 4 * time.Hour
```

Comment:
This resource would fetch the crd once every 4 hours. 🚀

Reply:
Thanks that TTL makes sense to me. If we need to refresh it sooner can we just restart the operator?

Reply:
@rossf7 Yes, restarting the pod will load CRDs from GitHub again.